src/OmniSharp.WebAssembly/CompilerLoggerPathRemapper.cs

```diff
@@ -24,6 +26,51 @@ public CompilerLoggerPathRemapper(IServiceProvider serviceProvider)
 
     public string Remap(string path)
     {
-        return path.Replace(_workspaceBasePath, _compilerLogBasePath);
+        return Remap(path, _workspaceBasePath, _compilerLogBasePath);
+    }
+
+    public static string Remap(string path, string workspacePath, string compilerLogBasePath)
```

Comment:
haven't really tested on unix (or done a lot of validation on this).  Might not be right, but hopeuflly describes the problems well